content/SCALE/DataProtection/Replication.md

```diff
@@ -0,0 +1,192 @@
+---
+title: "Replication"
+weight: 100
+---
+
+## Remote Replication
+
+
+{{< toc >}}
+
+Configure SSH in TrueNAS before creating a remote replication task. This ensures that both systems can connect to each other and new snapshots are regularly available for replication.
```

Comment:
We don't require both systems to be able to connect to each other. Local system connects to the remote system, not the other way around.

Reply:
fixed